addon/-private/state-machine/-base.ts

```diff
@@ -18,7 +23,7 @@ export default abstract class StateMachine extends EmberObject {
    * @property {A} stepTransitions
    * @private
    */
-  stepTransitions: Ember.MutableArray<string> = A();
+  stepTransitions: Ember.MutableArray<any> = A();
```

Comment:
This type change seems suspect... When is the array not populated by strings?

Reply:
It always is. I have been trying to reset it by assignment, which was throwing a type error. Instead I'll call `clear()`